governance.md

```diff
@@ -1,15 +1,104 @@
+* Perform code reviews on other's pull requests.
+* Regularly triage GitHub issues.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* Monitor OAM mailing list, slack channel and response to social media interactions.
+* Regularly attend the community meetings.
+
+The current list of maintainers is published and updated in `OWNERS.md` of every project.
+
+### Becoming a maintainer
+
+To become a maintainer of [OAM specification](https://github.com/oam-dev/spec): 
+- OAM specification is maintained by the OAM steering committee members **only**. Please check the [requirements of becoming steering committee member](#Becoming-a-Steering-Committee-Member).
+
+
+To become a maintainer of other projects under `oam-dev` organization:
+
+- Active contributions for at least 1 month.
+- Get at least 5 non-trivial pull requests merged to the codebase.
+- Primary reviewer for at least 3 non-trivial pull requests from others.
+- Done through PR to update the project’s OWNERS file.
+- Self-nominate or be nominated by existing maintainers in the project.
+- With no objections from existing maintainers in the project.
+
+### Removing a maintainer
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. Once volunteered, it will automatically take effect unless objection is raised from existing maintainers. In extreme cases removing maintainers can also occur by a vote of steering committee.
+
+
+### Github Project Administration
+
+Maintainers will be added to the OAM GitHub organization (if they are not already) and added to corresponding GitHub maintainers team. The maintainers team is per project.
+
+## Voting
+
+In general, we prefer that technical issues be resolved at project level by project maintainers with "non-objection and negotiation" approach. But if a essential dispute cannot be decided, the steering committee can be called in to decide an issue. If the steering committee members themselves cannot decide an issue, the issue will be resolved by voting. 
+
+- All changes in governance, additions and removals of steering committee members require a **2/3 majority**, while other decisions and changes require only a simple majority.
+- Only steering committee members can vote.
+- Organization voting:
+    - Each organization (regardless of the number of steering committee members associated with or employed by that company/organization) receives **one** organization vote.
+    - [Nonaffiliated Individual](#Nonaffiliated-individual) receives one organization vote. 
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR, and a link to that issue or PR added to the maintainers meeting agenda document. Voters should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
```

Comment:
All voting, should be recorded in a "decision record" type of file. This way, it can serve as a reference later. Otherwise, the voted decisions might end up being short lived.